anda/apps/flatpost/flatpost-mime.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="application/vnd.flatpak.ref">
```

Comment:
Since you're using `application/vnd.flatpak.ref`, you need to edit the desktop file to include `MimeType=application/vnd.flatpak.ref`

Reply:
I added a patch to the .desktop file and opened a PR upstream https://github.com/GloriousEggroll/flatpost/pull/6. Here's the new .desktop file.
```
$ cat /tmp/usr/share/applications/com.flatpost.flatpostapp.desktop 
[Desktop Entry]
Name=Flatpost
Exec=python3 /usr/bin/flatpost
Icon=com.flatpost.flatpostapp
Type=Application
Categories=Utility;
MimeType=application/vnd.flatpak.ref;application/vnd.flatpak.repo;
```